pertpy/tools/_mixscape.py

```diff
@@ -194,7 +194,9 @@ def mixscape(
         if layer is not None:
             X = adata_comp.layers[layer]
         else:
-            X = adata_comp.X
+            warning = "Using .layers[\"X_pert\"] as default!"
+            print(warning)
```

Comment:
Why make this print so complex? Also, if this is the default behavior there's no need for the warning.

Reply:
Good point